source/frontend/src/models/generate/GenerateOwner.ts

```diff
@@ -37,4 +42,43 @@ export class Api_GenerateOwner {
           .filter(x => !!x)
           .join(' ');
   }
+
+  static fromApiPerson(model: Api_Person): Api_GenerateOwner {
+    let generateOwner = new Api_GenerateOwner(null);
+    generateOwner.given_name = model.firstName ?? '';
+    generateOwner.last_name_or_corp_name = model.surname ?? '';
+    generateOwner.other_name = model.middleNames ?? '';
+    generateOwner.incorporation_number = '';
+    generateOwner.registration_number = '';
+    generateOwner.address = new Api_GenerateAddress(getApiPersonOrOrgMailingAddress(model) ?? null);
+    generateOwner.is_corporation = false;
+    generateOwner.owner_string = [
+      generateOwner.given_name,
+      generateOwner.last_name_or_corp_name,
+      generateOwner.other_name,
+    ]
+      .filter(x => !!x)
+      .join(' ');
+
+    return generateOwner;
+  }
+
+  static fromApiOrganization(model: Api_Organization): Api_GenerateOwner {
+    let generateOwner = new Api_GenerateOwner(null);
+
+    generateOwner.given_name = '';
+    generateOwner.last_name_or_corp_name = model.name ?? '';
+    generateOwner.other_name = model.alias ?? '';
+    generateOwner.incorporation_number = model.incorporationNumber ?? '';
+    generateOwner.registration_number = '';
+
+    const mailingAddress = model.organizationAddresses?.find(
+      addr => addr?.addressUsageType?.id === AddressTypes.Mailing,
+    );
```

Comment:
nit: I believe **getApiPersonOrOrgMailingAddress()** already supports either a person or an org so why not use that method instead of repeating that logic here?